src/presentation/http/middlewares/note/useNoteResolver.ts

```diff
@@ -40,6 +50,19 @@ export default function useNoteResolver(noteService: NoteService): {
     }
   }
 
+  /**
+   * Search for Note by public id in passed body in upload file request and resolves a note by it
+   *
+   * @param requestData - fastify request data. Can be query, params or body
+   */
+  async function resolveNoteByMultipartValueInUploadFileBody(requestData: FastifyRequest['body']): Promise<Note | undefined> {
+    if (hasProperty(requestData, 'notePublicId') && notEmpty(requestData.notePublicId)) {
+      const publicId = requestData.notePublicId as MultipartValue<NotePublicId>;
+
+      return await noteService.getNoteByPublicId(publicId.value);
```

Comment:
do we really need this middleware? It seems like a rear-case. I think, if it is used only in a single router, its better to call `noteService.getNoteByPublicId()` directly in a route hander

Reply:
done